package-lock.json

```diff
@@ -1,8643 +1,8 @@
 {
   "name": "editorjs-toggle-block",
-  "version": "0.3.12",
-  "lockfileVersion": 2,
+  "version": "0.3.13",
+  "lockfileVersion": 1,
```

Comment:
Please let's keep the `package-lock.json` file with lockfileVersion 2. You probably need to update `npm`.